contrib/org-present/keybindings.el

```diff
@@ -0,0 +1,19 @@
+(add-hook 'org-present-mode-hook
+          (lambda ()
+            (org-present-big)
+            (org-display-inline-images)
+            (org-present-hide-cursor)
+            (org-present-read-only)
+            (define-key evil-normal-state-local-map (kbd "h") 'org-present-prev)
+            (define-key evil-normal-state-local-map (kbd "l") 'org-present-next)
+            (define-key evil-normal-state-local-map (kbd "q") 'org-present-quit)))
+
+(add-hook 'org-present-mode-quit-hook
+          (lambda ()
+            (org-present-small)
+            (org-remove-inline-images)
+            (org-present-show-cursor)
+            (org-present-read-write)
+            (define-key evil-normal-state-local-map (kbd "h") nil)
```

Comment:
@CestDiego Once the user presses `q`, the quit-hook removes the `h` key binding, so it should be back to a normal org buffer.

However, I'm open to other key bindings :)